### DISCUSSION TITLE
Is there any way to disable built-in extensions using a file?

### DISCUSSION THREAD

**Discussion by daher13** [VSCodium/vscodium#823] [Q&A] [answered]
I'm trying to figure out some way to disable built-in extensions using a file in order to backup the configuration files later and automatically disable the built-in extensions.

**Comment by daiyam** [comment]
@daher13 In the gallery, you can filter by `Built-In` and then you can disable any extension/theme or language you want.

**Comment by daiyam** [comment, marked as answer]
For syncing your config, I've made the extension Sync Settings [OpenVSX](https://open-vsx.org/extension/zokugun/sync-settings) [VSMarketplace](https://marketplace.visualstudio.com/items?itemName=zokugun.sync-settings).

**Reply by daher13** [reply]
Thank you very much! I will use it!

**Comment by GitMensch** [comment]
So to answer the questions:

* No, it is not possibly to do so via a file, only in the settings directly (so maybe the answer is "yes - if you copy your complete ~/.vscodium folder, then you have also "backed up" that setting).
* Yes, it is possible to save and restore your configurations - by using another extension as noted by @daiyam. A backup would then mean to install this extension, then configure it,then use its download option.

**Reply by daher13** [reply]
The solution, without the extension was save the file .config/VSCodium/User/globalStorage/state.vscdb.
Then, when restore, the built-in extensions will keep disabled as you configured. Thank you!
